src/main/java/core/mate/academy/service/MachineProducerBulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MachineProducerBulldozer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> bulldozers = new ArrayList<>();
+        bulldozers.add(new Bulldozer());
```

Comment:
It is actual for all models. Add custom fields to your bulldozer according to the [task](https://github.com/BolsunovaNataliia/jv-pecs/blob/jv-pecs-solution/src/main/java/core/mate/academy/model/Bulldozer.java#L4).